plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -387,7 +473,7 @@ public void killContainer(String container, int signal) throws IOException {
      * @throws IOException
      */
     public void killContainer(String container) throws IOException {
-        killContainer(container, 9);
+        killContainer(KillContainerParams.from(container));
```

Comment:
So why do you want to set default behavior in API client instead of letting API server do this